workflows/scripts/merge_logs.py

```diff
@@ -123,7 +123,7 @@ def combine_logs(logs: list[str], outname: str) -> None:
     dir_path = f"logs/{id_tag}-archived_logs"
 
     # -- creating directory for archiving logs
-    archive_dir_obj = Path(dir_path)
+    archive_dir_obj = pathlib.Path(dir_path)
```

Comment:
Seeing the use of archiving here made me wonder if [logger log rotation techniques](https://docs.python.org/3/library/logging.handlers.html#baserotatinghandler) might be useful. No changes required, bringing it up just in case.